tchannel/tornado/peer.py

```diff
@@ -453,6 +453,9 @@ def __init__(self,
         self.tchannel = peer_group.tchannel
         self.service = service
         self.parent_tracing = parent_tracing
+        if self.tchannel.get_context():
+            self.parent_tracing = (parent_tracing or
+                                   self.tchannel.get_context().parent_tracing)
```

Comment:
nit: could be cleaner

    if not self.parent_tracing and self.tchannel.get_context():
        self.parent_tracing = self.tchannel.get_context().parent_tracing

Also, I'm not sure if `get_tracing` needs to exist. what's the use case for people needing to access tho context object?